Code/GraphMol/GeneralizedSubstruct/catch_tests.cpp

```diff
@@ -116,6 +116,34 @@ TEST_CASE("tautomer basics") {
   }
 }
 
+TEST_CASE("tautomer error") {
```

Comment:
Only part of this because of the change in the PKL output that I asked you about.  THis was part of trying to understand what happeded to that.  I have removed this test